.github/workflows/lint-test.yml

```diff
@@ -32,6 +32,25 @@ jobs:
     - name: Run pre-commit
       uses: pre-commit/action@v2.0.3
 
+  docs:
+    # We want to run on external PRs, but not on our own internal PRs as they'll be run
+    # by the push to the branch. Without this if check, checks are duplicated since
+    # internal PRs match both the push and pull_request events.
+    if:
+      github.event_name == 'push' || github.event.pull_request.head.repo.full_name !=
+      github.repository
+    runs-on: ubuntu-20.04
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

Reply:
this is intentional, but should be commented why.

I was mirroring the rtd workflow as much as possible

Reply:
https://github.com/DisnakeDev/disnake/pull/564/commits/e01ba2f89faf6104604cd803f9d3e9b8f12ef7a4